app/Http/Controllers/Auth/RegisterController.php

```diff
@@ -62,6 +62,14 @@ protected function create(StoreTenant $request)
 		$tenant->setOwner($user);
 		$tenant->owner->notify(new ActivateTenant);
 
+		$payStackCustomer = PayStack::createCustomer([
```

Comment:
this [method](https://github.com/unicodeveloper/laravel-paystack/blob/master/src/Paystack.php#L393) dosent take an array. (i think it should).

we should able to merge the request object with the expected properties like so

```suggestion
                $request->merge([
                'fname' => $user->firstname,
                'lname' => $user->lastname,
                'email' => $user->email,
                ]);
		$payStackCustomer = PayStack::createCustomer();
```

Reply:
I understand now. With the $request->merge method, the new user information will be available in the request data under the paystack variables names (fname, lname, email). But doing it like that, it exposes the password also in the request. I know the password is encrypted though. I think the paystack method should be able to accept data.

Reply:
dont worry about the other variables , ultimatlly they will be ignored , not that im merging the current user that was created above instead of using the data in the request object. Just because i want to be explicit as to what variable is the source of truth

Reply:
Okay. That makes sense. I have resolved the issue. Committed and pushed the changes